pom.xml

```diff
@@ -3,43 +3,37 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.23</version>
+        <version>4.16</version>
         <relativePath />
     </parent>
     <properties>
+        <revision>1.193</revision>
+        <changelist>-SNAPSHOT</changelist>
+        <gitHubRepo>jenkinsci/sauce-ondemand-plugin</gitHubRepo>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-        <jenkins.version>1.609.2</jenkins.version>
+        <jenkins.version>2.235.5</jenkins.version>
         <!-- Java Level to use. Java 7 required when using core >= 1.612 -->
         <java.level>8</java.level>
-        <plugins.credentials.version>2.1.1</plugins.credentials.version>
-        <plugins.matrix-project.version>1.4</plugins.matrix-project.version>
-        <plugins.maven-project.version>2.11</plugins.maven-project.version>
         <plugins.run-condition.version>1.0</plugins.run-condition.version>
-        <plugins.junit.version>1.21</plugins.junit.version>
-        <plugins.workflow.version>1.15</plugins.workflow.version>
         <ci-sauce.version>1.152</ci-sauce.version>
         <saucerest.version>1.0.42</saucerest.version>
     </properties>
 
 
     <artifactId>sauce-ondemand</artifactId>
-    <version>1.193-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
     <name>Jenkins Sauce OnDemand plugin</name>
-    <url>http://wiki.jenkins-ci.org/display/JENKINS/Sauce+OnDemand+Plugin</url>
+    <url>https://github.com/jenkinsci/sauce-ondemand-plugin</url>
 
     <scm>
-        <connection>scm:git:git://github.com/jenkinsci/sauce-ondemand-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:jenkinsci/sauce-ondemand-plugin.git</developerConnection>
-        <tag>HEAD</tag>
+        <connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+        <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+        <url>https://github.com/${gitHubRepo}</url>
+        <tag>${scmTag}</tag>
     </scm>
 
     <developers>
-      <developer>
-        <id>halkeye</id>
```

Comment:
fyi @halkeye pretty sure you're fine with this

Reply:
yea, they removed me a long time ago
I'm pretty sure yeh doesn't work there anymore either.